src/supportability/requestLog.ts

```diff
@@ -0,0 +1,88 @@
+import {Request, Response} from 'express'
+import {BFFException} from '../lib/exceptions';
+
+export class RequestLog {
```

Comment:
It's probably not crucial in release 1, but we should think of using some logging interface, which people can hook into. This way it will be possible for someone to log, e.g. directly to a remote system, etc.